### DISCUSSION TITLE
[World Boss] Old Lord Kazzak loot

### DISCUSSION THREAD

**Discussion by Hogofogo1** [chromiecraft/chromiecraft#5965] [Suggestions]
Could we have an option to obtain currently unobtainable loot from the old world boss Lord Kazzak? 
Implementation such as addition to the current Doomlord Kazzak loot table as an extra 1-2 items per kill seems appropriate 
as the gear is now irrelevant as far as stats boosts go.

**Comment by BeefSteaker** [comment]
Highlord Kruul took over in blasted lands and should have the same loot.

Are there any items in particular that are missing?

**Reply by Hogofogo1** [reply]
Afaik he was replaced by Doom lord Kazzak and is no longer present in Azeroth
https://wowpedia.fandom.com/wiki/Highlord_Kruul

**Reply by BeefSteaker** [reply]
Other way around. After the dark portal event, kazzak moved to outland and kruul replaced him in azeroth

**Reply by Hogofogo1** [reply]
Were there any sightings since tbc release?

**Reply by Hihilist** [reply]
Kruul replaced Kazzak as it is written in this news https://www.chromiecraft.com/en/the-burning-crusade-on-chromiecraft/.
But it was not a permanent replacement and only for some time-gated window. After that, neither Kruul nor Kazzak are available, making their loot unobtainable.

**Reply by Tehmirror** [reply]
There is one item in particular that is still relevant from the old loot table: Blazefury Medallion is pretty strong for Arms and 1h Fury. I would really like it if there was a way to get this item for people like me who changed their mains in tbc or did not raid in classic.

**Reply by Hogofogo1** [reply]
Yea, I'd like to experiment with the medallion as well

**Comment by Hogofogo1** [comment]
Bump
Any possibility of implementation from dev team?

**Comment by Amrx2** [comment]
Hi there, as Beef has mentioned above we are not looking to reintroduce vanilla Kazzak given their presence in Outlands now. We won't be adding loot to other mobs as a replacement either given alternatives exist freely and this does not hamper player progression.

Once our PTR is up and running you can additem the medallion as well should you wish to see its effects.

Cheers,
Construct

**Reply by Wheemix** [reply]
I know this is an old thread, sorry, but saying "this does not hamper player progression" is not really correct when his loot is literal BiS for all of TBC for fury warriors (who definitely cannot afford losing even more dps as already one of the weakest specs in the game right now).

Besides that, it also just feels like removing old content from the game for no reason, is anything gained by making him not spawn when he was already implemented and all?

**Reply by heyitsbench** [reply]
> [...] is anything gained by making him not spawn [...]

Blizzlike not to have him around. Though I personally would very much like Kazzak and Ony60 available during current/future content, if nothing else to be able to grab transmogs.

**Reply by Wheemix** [reply]
I expected this to be what it comes down to but it is not as clear an answer as it may seem. This server with the unusual situation it is in has always had to balance between different kinds of blizzlike, as it was intended, as it was in vanilla, as it was in TBC, as it was in 3.3.5 and now even classic on top of everything. So it is not like all the hands are tight and nothing can be done, a decision was taken that removing these bosses is the better choice and I am just here trying to provide counter arguments and a plea for helping poor fury nerds :)

**Comment by Anwyyn** [comment]
Bring Highlord Kruul back plsss
